eudemo/eudemo/tf_coils/tf_coils.py

```diff
@@ -661,8 +662,17 @@ def _build_xyz_case(
 
         # Cut away straight sweep before fusing to protect against degenerate faces
         # Keep the largest piece
-        pieces = boolean_cut(solid, inboard_casing)
-        solid = max(pieces, key=lambda x: x.volume)
+        try:
+            pieces = boolean_cut(solid, inboard_casing)
+            solid = max(pieces, key=lambda x: x.volume)
+        except Exception as e:  # Part.OCCError :/
```

Comment:
We should probably have a wrapper on boolean cut so that it returns a bluemira error. I think we wrote a decorator to do that at some point. I will see if I can find it. 

Just a note don't need to do it now

Reply:
Alright, I've hopefully fixed this by finding a better way of making the casing geometry.